tests/khmer_tst_utils.py

```diff
@@ -219,3 +220,8 @@ def longify(listofints):
     if sys.version_info.major < 3:
         return map(long, listofints)  # pylint: disable=bad-builtin
     return listofints
+
+def copy_test_data(testfile):
+    infile = get_temp_filename('test.fq')
```

Comment:
Hardcoding test.fq is problematic -- if the function is run multiple times, it will get overwritten, and it may also not be an fq file. You could replace 'test.fq' with os.path.basename(testfile), which would strip off the directory name but leave the filename itself intact.